src/adapters/mc/OsConfigResource.c

```diff
@@ -927,15 +964,10 @@ void MI_CALL OsConfigResource_Invoke_SetTargetResource(
 
     if (NULL == g_mpiHandle)
     {
-        if (!RefreshMpiClientSession())
-        {
-            miResult = MI_RESULT_FAILED;
-            mpiResult = ESRCH;
-            LogError(context, miResult, GetLog(), "[OsConfigResource.Set] Failed starting the MPI server (%d)", mpiResult);
-        }
+        RefreshMpiClientSession();

```

Comment:
We do not want to log anymore if this fails?

Reply:
It wasn't just logging, it was failing the Set call. That was removed because now we have a fallback net.